spi: Support display qspi data command

ThanhNguyenRVC opened the pull request:
Change includes:
- Support command "XRS" for display qspi data

vudangRVC commented on dgmodul4.c:
```diff
@@ -696,6 +699,89 @@ void dgClearSpiflash0(void)
+	FUNCTION		: QSPI memory read			                *
+	COMMAND			: XRS                                       *
+	INPUT PARAMETER	: XRS  {sadr {eadr}}  		                *
+*****************************************************************/
```
@ThanhNguyenRVC ,

Check asterisk at line 703, 704, 705, 706

ThanhNguyenRVC replied:
It been fixed at [bf8a0b0](https://github.com/vudangRVC/flash-writer-sst/commit/bf8a0b070a493f82a7c3761e40b09f081ee442a1)

vudangRVC commented on dgmodul4.c:
```diff
@@ -696,6 +699,89 @@ void dgClearSpiflash0(void)
+		byte_count = 0x0;
+
+		/********* Dump Data 1Line All Display *********************/
+		if (!((qspiEndAdd&0x0000000F) == 0x0000000F))
```
Check the space of "qspiEndAdd&0x0000000F"

ThanhNguyenRVC replied:
It been fixed at [bf8a0b0](https://github.com/vudangRVC/flash-writer-sst/commit/bf8a0b070a493f82a7c3761e40b09f081ee442a1)

vudangRVC commented on dgmodul4.c:
```diff
@@ -696,6 +699,89 @@ void dgClearSpiflash0(void)
+				}
+				PutStr(buf, 0);
+			}
+			if ((byte_count & 0xF) == 0xF)
```
This "if" condition is used to display data in ascii, right?
Please add a comment for it the same as for "Dump Data Display blank" or "Dump Address Display"

ThanhNguyenRVC replied:
It been fixed at [bf8a0b0](https://github.com/vudangRVC/flash-writer-sst/commit/bf8a0b070a493f82a7c3761e40b09f081ee442a1)

ThanhNguyenRVC committed:
renesas: spi: Support the XRS command

Change includes:
    - Add XRS command to support QSPI display data
	- Remove hardcode for DDR, update makefile for support board defined
    - Indent code for better readability

Signed-off-by: Thanh Nguyen <thanh.nguyen.wx@bp.renesas.com>
